alembic/versions/5d4ec1330a98_entry_lock.py

```diff
@@ -0,0 +1,38 @@
+"""Entry lock
+
+Revision ID: 5d4ec1330a98
+Revises: 245914e1ddf9
+Create Date: 2022-08-29 10:24:39.968959
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = '5d4ec1330a98'
+down_revision = '245914e1ddf9'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('journal_entries', sa.Column('locked_by', sa.String(), nullable=True))
+    op.create_index(op.f('ix_journal_entries_locked_by'), 'journal_entries', ['locked_by'], unique=False)
+    op.create_unique_constraint('uq_journal_entries_id_locked_by', 'journal_entries', ['id', 'locked_by'])
```

Comment:
`id` is primary key so it's already unique. This constraint doesn't add anything.